cxbox-core/src/main/java/org/cxbox/core/dao/impl/MetadataUtils.java

```diff
@@ -465,5 +475,36 @@ private boolean isElementCollectionField(Root<?> root, String fieldName) {
 				);
 	}
 
+	private boolean isElementCollectionFieldFromPath(Root<?> root, String fullPath) {
```

Comment:
please add java doc. old core code was writted during project development - so java doc for old code is bad, but new code should be documented